use_case_examples/cifar_brevitas_training/evaluate_torch_cml.py

```diff
@@ -97,9 +87,15 @@ def main(args):
     # Load weights
     model.load_state_dict(checkpoint["state_dict"], strict=False)
 
-    # Create a representative input-set that will be used used for both computing quantization
-    # parameters and compiling the model
-    input_set = torch.randn(1, 3, 32, 32)
+    # Load the training set
+    train_set = get_train_set(dataset="CIFAR10", datadir=".datasets/")
+
+    # Create a representative input-set from the training set that will be used used for both
+    # computing quantization parameters and compiling the model
+    num_samples = 100
+    input_set = torch.stack(
+        [train_set[index][0] for index in range(min(num_samples, len(train_set)))]
+    )
```

Comment:
Is the dataset seeded? Let's make sure we have the same calibration set so that we don't include randomness in the calibration.

Reply:
it has never been seeded in any use case if I recall, not sure we want to start to do that, especially since this inputset is probably less "random" than using the initial `torch.randn(1, 3, 32, 32)` 😅

Reply:
Yes that torch.randn was not great but the calibration data point was not having any impact on accuracy. Now it does so maybe we want to stick with one accuracy to be able to see any other causes of drop (else than random).

Reply:
I don't think it matters a lot, the accuracy is stable with a 100 samples normalized between -1 and 1. Besides, we already have an issue for seeding notebooks : https://github.com/zama-ai/concrete-ml-internal/issues/1713